PocketKit/Sources/PocketKit/LoggedOut/LoggedOutViewModel.swift

```diff
@@ -36,6 +38,23 @@ class LoggedOutViewModel: ObservableObject {
         self.networkPathMonitor = networkPathMonitor
 
         networkPathMonitor.start(queue: DispatchQueue.global())
+        currentNetworkStatus = networkPathMonitor.currentNetworkPath.status
+        networkPathMonitor.updateHandler = { [weak self] path in
+            DispatchQueue.main.async {
+                self?.updateStatus(path.status)
+            }
+        }
+    }
+
+    private func updateStatus(_ status: NWPath.Status) {
+        guard status != currentNetworkStatus else {
+            return
+        }
+
+        if currentNetworkStatus == .unsatisfied, status == .satisfied, presentOfflineView == true {
```

Comment:
Curious if we actually need all these checks here?

If `presentOfflineView` is already false, would setting it to false again have any effect?